psydac/fem/tensor.py

```diff
@@ -418,52 +435,56 @@ def compute_interpolant( self, values, field ):
 
     def reduce_grid(self, axes=[], knots=[]):
```

Comment:
Default argument values must be immutable. Please use empty tuples `()`, or set them to `None`.